apple/testing/default_runner/ios_sample_test_runner.template.sh

```diff
@@ -45,6 +45,12 @@ SDK_VERSION="$(xcrun --sdk iphonesimulator --show-sdk-version)"
 
 NEW_SIM_ID=$(xcrun simctl create "$RANDOM_NAME" "iPhone 6" "$SDK_VERSION")
 
+# Clean our simulator up even if we fail along the way
+function cleanup {
```

Comment:
so trap doesn't stack, which means that the one in line 33 would be discarded... would you mind moving this to the top and merge it with the `rm -rf` part? this would also mean creating the simulator at the top too